commitlint.config.ts

```diff
@@ -558,7 +560,28 @@ module.exports = {
                         `Please use full URLs instead of #XYZ refs.`
                     ];
                 },
+    
+                'proper-revert-message': ({header, body}: {header: any, body: any}) => {
+                    let offence = false;
+
+                    if (header.match(/^[Rr]evert ".+"$/) !== null) {
+
+                        // does msg have a body?
+                        if (body !== null) {
+                            let bodyStr = convertAnyToString(body, "body").trim();
+                            let lines = bodyStr.split('\n');
+                            offence = lines[0].match(/^This reverts commit [^ ]{40}.$/) !== null && lines.length == 1;
```

Comment:
@realmarv are you sure you don't need to escape the '.' here? AFAIU, dots in regexes act as any character, but here I think you're trying to match it with a real dot

Reply:
Done